locale/Makefile

```diff
@@ -6,20 +6,19 @@
 # make tx-update - download and merge translations from Transifex
 # make clean - clean everything
 #
-DOMAIN = katello
-VERSION = $(shell git describe --abbrev=0 --tags)
+DOMAIN = $(shell ruby -rrubygems -e 'puts Gem::Specification::load(Dir.glob("../*.gemspec")[0]).name')
+VERSION = $(shell ruby -rrubygems -e 'puts Gem::Specification::load(Dir.glob("../*.gemspec")[0]).version')
 POTFILE = $(DOMAIN).pot
 MOFILE = $(DOMAIN).mo
-ACTIONFILE = action_names.rb
 POFILES = $(shell find . -name '$(DOMAIN).po')
 MOFILES = $(patsubst %.po,%.mo,$(POFILES))
 POXFILES = $(patsubst %.po,%.pox,$(POFILES))
 EDITFILES = $(patsubst %.po,%.edit.po,$(POFILES))
+JSFILES = $(shell find ../app/assets/javascripts/*/locale -name '$(DOMAIN).js')
 
 %.mo: %.po
 	mkdir -p $(shell dirname $@)/LC_MESSAGES
 	msgfmt -o $(shell dirname $@)/LC_MESSAGES/$(MOFILE) $<
-	touch $(shell dirname $@)/LC_MESSAGES/$(MOFILE)
```

Comment:
I have noticed that `msgfmt` doesn't always update the timestamp, if there are no changes. Now I'm unsure if this is still an issue after the time_stamp handling is fixed.